components/selectors/lib.rs

```diff
@@ -7,9 +7,11 @@
 
 #[macro_use] extern crate bitflags;
 #[macro_use] extern crate cssparser;
+#[macro_use] extern crate heapsize_derive;
```

Comment:
IIRC we initially made Stylo not depend on heapsize, but now it’s included in Firefox anyway through WebRender, so shrug.